src/blueTheme.ts

```diff
@@ -6,88 +6,88 @@ All rights reserved.
-            onPrimary: {
-                light: black[500],
-                main: white[50],
-                dark: white[50],
-            },
-            disabled: Color(black[500]).alpha(themeOpacity.disabled).string(),
-        },
-        actionPalette: {
-            active: gray[500],
-            background: white[500],
-            disabled: Color(black[500]).alpha(themeOpacity.disabled).string(),
-            disabledBackground: Color(black[200]).alpha(themeOpacity.disabledBackground).string(),
-        },
-        disabled: Color(black[500]).alpha(themeOpacity.disabled).string(),
-        notification: lightBlue[500],
-        opacity: themeOpacity,
-        overrides: {
-            snackbar: {
-                accent: blue[200],
-            },
-            bottomNavigation: {
-                inactive: blue[200],
-            },
-            button: {
-                contained: {
-                    text: {
-                        disabled: blue[200],
-                    },
-                },
-            },
+        ...MD3LightTheme.colors,
+        primary: BLUIColors.primary[40],
+        onPrimary: BLUIColors.primary[100],
+        primaryContainer: BLUIColors.primary[80],
+        onPrimaryContainer: BLUIColors.primary[30],
+        secondary: BLUIColors.primary[40],
+        onSecondary: BLUIColors.primary[100],
+        secondaryContainer: BLUIColors.primary[80],
+        onSecondaryContainer: BLUIColors.primary[30],
+        tertiary: BLUIColors.primary[40],
+        onTertiary: BLUIColors.primary[100],
+        tertiaryContainer: BLUIColors.primary[80],
+        onTertiaryContainer: BLUIColors.primary[30],
+        error: BLUIColors.error[40],
+        errorNonText: BLUIColors.error[50],
+        onError: BLUIColors.primary[100],
+        errorContainer: BLUIColors.error[90],
+        onErrorContainer: BLUIColors.error[30],
+        background: BLUIColors.neutral[98],
+        onBackground: BLUIColors.neutral[10],
+        surface: BLUIColors.neutral[98],
+        onSurface: BLUIColors.neutral[10],
+        surfaceVariant: BLUIColors.neutralVariant[90],
+        onSurfaceVariant: BLUIColors.neutralVariant[30],
+        outline: BLUIColors.neutralVariant[50],
+        outlineVariant: Color(BLUIColors.neutralVariant[50]).alpha(0.25),
+        shadow: BLUIColors.primary[0],
+        scrim: BLUIColors.primary[0],
+        inverseSurface: 'rgb(47, 48, 51)',
+        inverseOnSurface: 'rgb(241, 240, 244)',
+        inversePrimary: 'rgb(169, 199, 255)',
+        elevation: {
+            level0: 'transparent',
+            level1: BLUIColors.primary[100],
+            level2: BLUIColors.neutral[97],
+            level3: BLUIColors.neutral[94],
+            level4: BLUIColors.neutral[92],
+            level5: BLUIColors.neutral[90],
         },
+        surfaceDisabled: 'rgba(26, 27, 30, 0.12)',
+        onSurfaceDisabled: 'rgba(26, 27, 30, 0.38)',
+        backdrop: 'rgba(45, 48, 56, 0.4)',
```

Comment:
`backdrop` is not part of the standard Material Design color.

Reply:
I can able to see `backdrop` variant in the React Native Paper theming  [doc](https://callstack.github.io/react-native-paper/docs/guides/theming/#:~:text=inversePrimary-,backdrop,-fonts%20(object)
Is this variant not part of our theme?

Reply:
Correct, it is not part of the Brightlayer UI theme. If `backdrop` refers to `scrim` with transparency, then Material Design prefers pure black with 0.32 opacity for both light and dark theme, I believe.

Reply:
Okay, I will remove this color variant from both light and dark theme